examples/browser/src/index.ts

```diff
@@ -26,24 +27,24 @@ import { Mesh, OrderEvent, SignedOrder, BigNumber } from '@0x/mesh-browser';
     // This order is for demonstration purposes only and is invalid. It will be
     // rejected by Mesh. You can replace it with a valid order.
     const order: SignedOrder = {
-        chainId: 1,
-        makerAddress: '0xa3eCE5D5B6319Fa785EfC10D3112769a46C6E149',
-        makerAssetData: '0xf47261b0000000000000000000000000e41d2489571d322189246dafa5ebde1f4699f498',
-        makerAssetAmount: new BigNumber('1000000000000000000'),
-        makerFee: new BigNumber('0'),
-        makerFeeAssetData: '0x',
+        signature:
```

Comment:
Is there a particular order that is being used here?

Reply:
Yeah I decided to just grab one from api.0x.org. The old one was invalid because it didn't have the correct signature.